### ISSUE TITLE
DB test for Count() does not fail although it should

### ISSUE THREAD

**Issue by anatheka** [clouditor/clouditor#1197]
### What happened?

[This test](https://github.com/clouditor/clouditor/blob/1e028371a8cfbc571d7db225dbffc4d4843c9e23/service/orchestrator/certificates_test.go#L616) should fail, if the DB Count() function does not get "id=?" ([count, err := svc.storage.Count(req.Certificate,  req.Certificate.Id](https://github.com/clouditor/clouditor/blob/1e028371a8cfbc571d7db225dbffc4d4843c9e23/service/orchestrator/certificates.go#L142)).

### Components

Orchestrator

### Relevant log output

```bash
2023-09-08 12:33:54.347 UTC [3571] ERROR: syntax error at or near "f5a" at character 53
2023-09-08 12:33:54.347 UTC [3571] STATEMENT: SELECT count(*) FROM "certificates" WHERE ccc58f10-9f5a-4ac5-94bb-988b63fe2f02
```
```

**Comment by oxisto** [comment]
Was this fixed by https://github.com/clouditor/clouditor/pull/1196?

**Comment by anatheka** [comment]
Fixed by #1196.
